doc/site/modules/core/fiber.markdown

```diff
@@ -8,12 +8,16 @@ A lightweight coroutine. [Here][fibers] is a gentle introduction.
 
 ### Fiber.**abort**(message)
 
-Raises a runtime error with the provided message:
+Raises a runtime error with the provided message.  `message` may also be an object.
```

Comment:
`message`, in fact, can be any Wren value, as far as I know.  See src/vm/wren_value.h, struct `ObjFiber` --- it has a `Value error`.  Would you be willing to update the suggested language to note that?

Reply:
You can click the +/- and propose changes... I'm fine with whatever here as long as it adds clarity.

Reply:
Would you, please?  My Wren time is currently taken up by the switch-statement PR.  I appreciate it!